plugin/src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java

```diff
@@ -843,19 +837,26 @@ public void registerCoreMembers() {
         // @Syntax copyblock [<location>/<cuboid>] [to:<location>] (remove_original)
         // @Required 1
         // @Stable unstable
-        // @Short Copies a block to another location, keeping all metadata.
+        // @Short Copies a block to another location, keeping all data.
         // @Author aufdemrand, David Cernat
         // @Group world
         //
         // @Description
-        // TODO: Document Command Details
+        // Copies a block or cuboid area to another location while keeping stored data such as items in chests and furnaces.
+        // This is also useful for copying signs, command blocks or anything that has custom data.
+        // Specifying the argument remove_original to remove the block that was copied, acting as a move/cut block.
         //

```

Comment:
docs in general here are misleading, as the command is currently not very capably implemented and needs an update before the docs can validly make these claims.